01-exploring-pegasus.ipynb

```diff
@@ -31,7 +31,7 @@
     "\n",
     "The Advantage system is distinct from all previous generations of D-Wave's quantum computers in being the first to employ a new architecture made possible by recent advances in QPU fabrication technology. \n",
     "\n",
-    "QPUs are lattices of interconnected qubits. While some qubits connect to others via couplers, QPUs are not fully connected. Instead, qubits interconnect in a topology: Chimera for the DW-2000Q and Pegasus for the Advantage.\n",
+    "QPUs are lattices of interconnected qubits. While some qubits connect to others via couplers, QPUs are not fully connected. Instead, qubits interconnect in a topology: Chimera for the D-Wave 2000Q and Pegasus for the Advantage.\n",
```

Comment:
Mention Zephyr/Advantage2?

Reply:
I think not quite yet. Let Pegasus have its full day in the sun.